FluentTerminal.App/Utilities/JumpListHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using FluentTerminal.Models;
+using Windows.UI.StartScreen;
+using System.Collections.Generic;
+
+namespace FluentTerminal.App.Utilities
+{
+    public static class JumpListHelper
+    {
+        public static async void Update(IEnumerable<ShellProfile> profiles)
+        {
+            try
+            {
+                if (JumpList.IsSupported())
+                {
+                    var jumpList = await JumpList.LoadCurrentAsync();
+                    jumpList.Items.Clear();
+                    foreach (var profile in profiles)
+                    {
+                        var item = JumpListItem.CreateWithArguments("JumpList:" + profile.Id.ToString(), profile.Name);
```

Comment:
We might have other JumpList items in the future. I would rename this to ShellProfile